libpod/runtime.go

```diff
@@ -1,13 +1,10 @@
 package libpod
```

Comment:
Should we break runtime.go into multiple files, container.go, image,go, pod.go  These will get rather large once we port all of likpod into them.

Reply:
Yeah, that sounds like a plan. It's a trivial change, I think we can make it in here.

Reply:
Fixed